images/assets/wait_on_database_migrations.sh

```diff
@@ -2,7 +2,7 @@
 
 echo "Checking for database migrations"
 while true; do
-  /usr/local/bin/pulpcore-manager showmigrations | grep '\[ \]'
+  /usr/local/bin/pulpcore-manager showmigrations | grep '\[ \]' &> /dev/null
```

Comment:
:+1: Really a good change.
I thought, I saw you having a django based solution to this. Is improving this script left for another day?

Reply:
Absolutely. We don't _need_ to do it here.